packages/shared/src/lib/dateFormat.ts

```diff
@@ -56,6 +57,43 @@ const publishTimeRelative = (
   return `${numYears} ${numYears === 1 ? 'year' : 'years'} ago`;
 };
 
+const publishTimeRelativeShort = (
```

Comment:
Probably vibe code to generate some tests? Just to ensure how it would look like for example, no decimal places are displayed, etc.

Reply:
There are tests for previous implementations that I updated for this one.

Reply:
But yeah I'll look

Reply:
Added tests 01c5ddc5ad167396d708ea66826a476bc6e904ba